### DISCUSSION TITLE
我想在两侧加点信息，如何修改？

### DISCUSSION THREAD

**Discussion by zhlx2835** [AlistGo/alist#2056] [Q&A]
没找到安装的目录和安装文件

**Comment by xhofe** [comment]
https://alist.nn.ci/config/global.html#customize-body

**Comment by mixpplus** [comment]
> 没找到安装的目录和安装文件

你好，添加云盘之后再修改云盘的配置信息时不会立刻生效，每次都是要重启服务才会生效，有没有更好的方式让修改生效，有没有相关的配置项来禁用缓存呢，谢谢！

**Reply by anwen-anyi** [reply]
你现在用的是什么版本？ 这个好像已经修复了吧？我这里目前是没有这个问题了

**Reply by mixpplus** [reply]
版本：3.2.0
我是修改已经添加过的云盘信息，然后点击编辑，例如修改一下`根文件夹id`, 修改完成之后如果不重新启动不会立刻生效的
